src/index.html

```diff
@@ -5,9 +5,332 @@
+        <img
+          src="./images/phone.svg"
+          alt="phone icon"
+          class="header__phone-icon"
+        >
+      </a>
+
+      <input
+        type="checkbox"
+        name="menu"
+        id="menu"
+        class="header__menu-input"
+      >
+      <label for="menu" class="header__menu-label"></label>
+
+      <div class="header__contacts">
+        <a href="tel:+12345555555" class="header__call">+1 234 555-55-55</a>
+        <a href="tel:+12345555555" class="header__order">call to order</a>
+      </div>
+
+      <h1 class="header__title">Miami Condo Kings</h1>
+
+      <nav class="header__navigation nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#home" class="nav__link">Home</a></li>
+          <li class="nav__item"><a href="#about-us" class="nav__link">About us</a></li>
+          <li class="nav__item"><a href="#gallery" class="nav__link">Gallery</a></li>
+          <li class="nav__item"><a href="#consultation" class="nav__link">Consultation</a></li>
+          <li class="nav__item"><a href="#testimonials" class="nav__link">Testimonials</a></li>
+          <li class="nav__item"><a href="#contact-us" class="nav__link">Contacts</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <section class="main__about-us about-us" id="about-us">
+        <h2 class="about-us__slogan">Miami best <br>Real estate <span class="about-us__emphasis">here</span></h2>
+        <div class="about-us__inf">
+          <h3 class="about-us__title">Over 50 years experience in love.</h3>
+          <p class="about-us__text">Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.</p>
+          <p class="about-us__text">The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists. Working directly with developers - so have more useful information about the profitable deals.</p>
+          <p class="about-us__text"><strong>We will never be late!<br>Guaranteed secure transaction - 100%</strong></p>
+        </div>
+      </section>
+
+      <section class="main__gallery gallery" id="gallery">
+        <h2 class="gallery__title">Gallery</h2>
+
+        <article class="gallery__article">
+          <div class="gallery__line gallery__line--first">
+            <a href="https://www.google.com/maps/place/%D0%9D%D0%BE%D1%80%D1%82-%D0%9C%D0%B0%D1%8F%D0%BC%D1%96-%D0%91%D1%96%D1%87,+%D0%A4%D0%BB%D0%BE%D1%80%D0%B8%D0%B4%D0%B0+33162,+%D0%A1%D0%BF%D0%BE%D0%BB%D1%83%D1%87%D0%B5%D0%BD%D1%96+%D0%A8%D1%82%D0%B0%D1%82%D0%B8+%D0%90%D0%BC%D0%B5%D1%80%D0%B8%D0%BA%D0%B8/@25.9289247,-80.2148991,13z/data=!3m1!4b1!4m5!3m4!1s0x88d9addb1b652001:0x525bd42381a2a4cf!8m2!3d25.9260794!4d-80.1819268">
```

Comment:
make it easy to read for now
```suggestion
            <a href="https://google.com">
```